docs/user/guides/access/wifi.md

```diff
@@ -1,27 +1,25 @@
 ---
-title: WI-FI
+title: Wi-Fi
 sidebar_position: 1
 ---
 
-# Wifi Hotspot
+If you have bought a DAppNode from our store or you installed DAppNode in a physical device with a Wi-Fi chip inside, you can directly connect to your DAppNodes UI by connecting your client device to the DAppNode's Wi-Fi hotspot.
 
-If you have bought a DAppNode or you have installed DAppNode in a physical device with Wi-fi, you can directly connect to your server's admin UI by connecting your client device to the DAppNode's wifi hotspot.
+### Connecting to your DAppNode Wi-Fi
```

Comment:
ugh i just remebered. the stylization changed so probably best to find and replace but `DAppNode` is now `Dappnode` unless referring to the logo then its all lower case
```suggestion
### Connecting to your Dappnode Wi-Fi
```

Reply:
Will search through all the docs and change that, good to know!

Reply:
i dont think well im 97% sure stuff like the SSID and anything in the code is still the same as it was. maybe i should change this one, just the default SSID to match the new stylization, roll it out when the rebranded dappmanager is published.